### PR TITLE
Add nvmrc

### PR DESCRIPTION
**Pull request by Neuramare**

### CODE REVIEW

**Comment by Neuramare** [comment]
It's this version compatible with vercel deploy?

**Comment by angelolamonaca** [comment]
> It's this version compatible with vercel deploy?

We ar eusing heroku, and yes, heroku uses nvm and thsi version is available

**Comment by Neuramare** [comment]
is v20.9.0 lts?

**Comment by angelolamonaca** [comment]
> is v20.9.0 lts?

yes, it is

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v20.9.0
```
Comment (angelolamonaca) [inline .nvmrc:1]: we should add a new line

Comment (Neuramare) [inline .nvmrc:1]: is possible to add json extension?

Comment (angelolamonaca) [reply .nvmrc:1]: we should not

**Pull request merged**